src/components/ec-popover/__snapshots__/ec-popover.spec.js.snap

```diff
@@ -1,81 +1,97 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EcPopover component should add the z-index level class if the level was given 1`] = `
-<fv-dropdown-stub
-  ariaid="ec-popover-1"
-  arrowoverflow="true"
-  container="body"
-  data-test="ec-popover"
-  popperclass="ec-popover ec-popover--level-30"
```

Comment:
this `ec-popover--level-30` property should be set, but we can't see it in the new snapshot

Reply:
Fixed!